### PR TITLE
Implement wildcard route for organization pages

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: studio/pages/org/_/[[...routeSlug]].tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  const { routeSlug, ...queryParams } = router.query
+  const queryString =
+    Object.keys(queryParams).length > 0
+      ? new URLSearchParams(queryParams as Record<string, string>).toString()
+      : ''
+
+  const urlRewriterFactory = (slug: string | string[] | undefined) => {
+    return (orgSlug: string) => {
+      if (!Array.isArray(slug)) {
+        return `/org/${orgSlug}/general`
```
Comment (alaister) [inline studio/pages/org/_/[[...routeSlug]].tsx:46]: Is there a reason not to forward on the `queryString` anymore?

Comment (joshenlim) [reply studio/pages/org/_/[[...routeSlug]].tsx:46]: this particular page doesnt have any support for queryString though - hence why i removed it

Comment (alaister) [reply studio/pages/org/_/[[...routeSlug]].tsx:46]: Yeah why not :) might save us in future if we add something that uses a query string

Comment (joshenlim) [reply studio/pages/org/_/[[...routeSlug]].tsx]: gotcha! just pushed :)

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**